src/Core/src/Handlers/SwipeItemMenuItem/SwipeItemMenuItemHandler.iOS.cs

```diff
@@ -118,7 +118,7 @@ public override void SetImageSource(UIImage? platformImage)
 
 					try
 					{
-						button.SetImage(resizedImage.ImageWithRenderingMode(UIImageRenderingMode.AlwaysTemplate), UIControlState.Normal);
+						button.SetImage(resizedImage.ImageWithRenderingMode(UIImageRenderingMode.Automatic), UIControlState.Normal);
```

Comment:
With this change, there are failing tests like `VerifySwipeViewApperance`. That's because the icon is no longer tinted with the text color.

Current behavior:
<img width="398" alt="image" src="https://github.com/user-attachments/assets/efd129a7-ffaf-4a3b-808f-98e95602d052" />

With the changes:
![image](https://github.com/user-attachments/assets/a145d1fe-de9d-4c71-9ed3-67d943754415)